src/react/databrowser/buckets/BucketList.tsx

```diff
@@ -56,9 +48,18 @@ export default function BucketList({
 
         Cell({ value: name }: { value: string }) {
           return (
-            <T.CellClick onClick={handleCellClicked(name)}>{name}</T.CellClick>
+            <T.CellLink
+              onClick={(event) => event.stopPropagation()}
+              to={`/accounts/${accountName}/buckets/${name}/objects`}
+            >
+              {name}
+            </T.CellLink>
           );
         },
+        cellStyle: {
+          flex: '1',
+          paddingLeft: '1rem',
```

Comment:
Pixels/rem?

Reply:
For pixel/rem consideration we will check that later on